docs/running.rst

```diff
@@ -48,3 +48,63 @@ Your server can then be run by the simple command:
     honcho start
     
 On Windows, the ``--beat`` option may not be supported.
+
+Production installation of Wooey utilizing NGINX and uWSGI
+----------------------------------------------------------
+
+1. add a domain name to ALLOWED_HOSTS in django_settings.py in the settings dir of your project
+2. specify a static dir you like in STATIC_ROOT- default is OK
+3. make sure the entire directory tree is chmodded and chgrouped appropriately for the webserver user..sqlite db needs to be writable by them, etc.
```

Comment:
Can you specify if the user is using sqlite.db -- sqlite is not a good thing for a web server to be running off so I'd rather not encourage it.